packages/metrics/src/features/registration/fhirUtils.ts

```diff
@@ -68,6 +68,19 @@ export type DECLARATION_STATUS =
   | 'REQUESTED_CORRECTION'
   | 'CERTIFIED'
 
+export type USER_ACTION =
+  | 'DOWNLOADED'
+  | 'ARCHIVED'
+  | 'ASSIGNED'
+  | 'LOGGED_IN'
+  | 'LOGGED_OUT'
+
+export interface IUserAuditBody {
+  bundle: fhir.Bundle
+  action: USER_ACTION
+  additionalData?: string
```

Comment:
Can we have this as a freeform object so `Record<string, any>` so that anyone storing or fetching events wouldn't know that internally we store them as a JSON string?